docs/guides/cloud/analytics/enterprise-reporting.mdx

```diff
@@ -4,46 +4,103 @@ sidebar_position: 30
 sidebar_custom_props: { 'new_label': true }
 ---
 
-:::info
+Enterprise Reporting is available within Cypress Cloud so that you can spot trends and anomalies,
+track usage, and govern your overall test program across all projects in your organization. Included
+is the ability to interact with visual cards that highlight usage, test metrics, test types, flake,
+build versions, and more. The raw data from all graphs presented in Cloud can be downloaded in a variety
+of formats. And if you prefer to programatically retrieve test data for use within your own analytics
```

Comment:
```suggestion
of formats. And if you prefer to programmatically retrieve test data for use within your own analytics
```

Elsewhere in this repo and on https://www.merriam-webster.com/dictionary/programmatically it's spelled with two m characters.